PDF_A/3b/6.8 Embedded files/verapdf-profile-6-8-t03.xml

```diff
@@ -2,18 +2,21 @@
 <profile xmlns="http://www.verapdf.org/ValidationProfile" flavour="PDFA_3_B">
     <details creator="veraPDF Consortium" created="2016-02-15T10:58:08.854+03:00">
         <name>ISO 19005-3:2012 - 6.8 Embedded files - AFRelationship</name>
-        <description>In order to enable identification of the relationship between the file specification dictionary and the content that is referring to it, 
-	a new (required) key has been defined and its presence (in the dictionary) is required.</description>
+        <description>In order to enable identification of the relationship between the file specification dictionary
+            and the content that is referring to it, the new (required) key AFRelationship of type Name has been
+            defined and its presence (in the dictionary) is required.</description>
     </details>
     <hash></hash>
     <rules>
         <rule object="CosFileSpecification">
             <id specification="ISO_19005_3" clause="6.8" testNumber="3"/>
-            <description>In order to enable identification of the relationship between the file specification dictionary and the content that is referring to it,
-			a new (required) key has been defined and its presence (in the dictionary) is required.</description>
+            <description>In order to enable identification of the relationship between the file specification dictionary 
+                and the content that is referring to it, the new (required) key AFRelationship of type Name has been 
+                defined and its presence (in the dictionary) is required.</description>
             <test>AFRelationship != null</test>
             <error>
-                <message>The file specification dictionary for an embedded file does not contain the AFRelationship key</message>
+                <message>The file specification dictionary for an embedded file does not contain the AFRelationship key 
+                or the AFRelationship key has no type Name</message>
                 <arguments/>
```

Comment:
The file specification dictionary for an embedded file does not contain the AFRelationship key or it has invalid value type